docs/snippets/book/GettingStarted.Tests/UnitTest1.cs

```diff
@@ -0,0 +1,6 @@
+namespace GettingStarted.Tests
+{
```

Comment:
Use file scoped namespace ?

Reply:
Torn on this. Not an easy way to demonstrate that that's what's happening in the examples so my concern is that people might copy/paste and be confused by the result.